docs/plugins_configuration.rst

```diff
@@ -190,17 +190,45 @@ definition of an "implementation" (see
            """My fancy implementation."""
            ...
 
-Lastly, our implementation should be exposed via our package's
-:file:`pyproject.toml` Poetry plugins section, using the ``smqtk_plugins`` key,
-as inherited from the base :class:`~smqtk_core.plugin.Pluggable` mixin, like
-the following:
+Lastly, our implementation should be exposed via our package's entrypoint
+metadata, using the "smqtk_plugins" namespace. This namespace value is derrived
+from the base :class:`~smqtk_core.plugin.Pluggable` mixin's
+``PLUGIN_NAMESPACE`` class property.
+Entry point metadata may be specified for a package either via the
+:func:`setuptools.setup` function, the `setup.cfg` file, or, when using poetry,
+a ``[tool.poetry.plugins."..."]`` section in the :file:`pyproject.toml` file.
+The a above are illustrated in the following:
```

Comment:
"derrived`" to "derived"

"The a above are illustrated in the following:" to "The above are illustrated in the following:"

Reply:
Thanks! Fixed in the latest amended commit.